angular-client/src/pages/charging-page/charging-page-mobile/charging-page-mobile.component.html

```diff
@@ -0,0 +1,25 @@
+<vstack style="width: 100%" spacing="20px">
+  <typography variant="xx-large-title" content="Argos"></typography>
+  <typography variant="x-large-title" [content]="time | date: 'HH:mm:ss'"></typography>
+  <typography variant="x-large-title" content="{{ time | date: 'MM/dd/yyyy' }}"></typography>
+
+  ==
+  <charge-state-display style="width: 100%" />
+
+  <hstack style="width: 100%">
+    <battery-status-display style="width: 100%" />
+    <latency-display style="width: 70%" />
+  </hstack>
+
+  <hstack style="width: 100%">
+    <charging-switch style="width: 50%; height: 20vh" />
+    <pack-temp style="width: 50%; height: 20vh" />
+  </hstack>
+  <battery-voltage-display style="width: 100%; height: 20vh" />
```

Comment:
mobile view needs to be totally fixed for these middle components, I'd start by trying to make the components easily resizable/ reactive, instead of hardcoding widths/ heights. However, this should be the last thing you focus on, after all the data hookup stuff is working
![image](https://github.com/Northeastern-Electric-Racing/Argos/assets/29521172/81334ba8-85e9-4495-a0a5-f945de321f7c)